db/migrate/20200412194408_create_optional_fields.rb

```diff
@@ -0,0 +1,11 @@
+class CreateOptionalFields < ActiveRecord::Migration[5.2]
+  def change
+    create_table :optional_fields do |t|
+      t.string :field
+      t.string :value
```

Comment:
this should probably be a `text` as conceptually is free form text

Reply:
In my thinking it's never more than a sentence (usually less) which is why it has a constraint of 128 characters (on the Model level). Text is usually for text longer than 255 characters, but I can change it to text here to make the intent clearer.

Reply:
Make sense, let's keep it a string :) (there's not much difference in PostgreSQL between varchar and text anyway)